lua/textcase/plugin/plugin.lua

```diff
@@ -274,7 +274,11 @@ function M.visual(case_method)
 
   if M.state.telescope_previous_visual_region ~= nil then
     utils.set_visual_region(M.state.telescope_previous_visual_region)
-    vim.api.nvim_feedkeys("gvg@", "i", false)
```

Comment:
`gv` reselect last selection. It was supposed to go back to visual selection. Why isn't it working 🤔 
I struggled to make this work, wondering why it isn't anymore

Reply:
I really couldn't figure out why it's not working. I looked into it and it seemed to be the right thing to do here.